modules/statics/src/base.ts

```diff
@@ -275,9 +275,14 @@ export enum CoinFeature {
   MPCV2 = 'mpcv2',
 
   /**
-   * This coin supports acceleration or nonce filling txn for stuck transactions
+   * This coin supports acceleration or nonce filling txn for stuck transactions for tss wallet
    */
-  STUCK_TRANSACTION_MANAGEMENT = 'stuck-transaction-management',
```

Comment:
Instead of removing this value I think it would be better to: 
- mark it as deprecated
- update usage internally to check either `STUCK_TRANSACTION_MANAGEMENT` or `STUCK_TRANSACTION_MANAGEMENT_ONCHAIN`


In later releases:
- remove `STUCK_TRANSACTION_MANAGEMENT`


this way its not a breaking change, at least not immediately.

Reply:
I added this feature last week and hasn't been used anywhere yet